app/routes/missions.fermi/route.mdx

```diff
@@ -3,9 +3,9 @@ handle:
   breadcrumb: Fermi
 ---
 
-import logo from './logo.svg'
+import logo from "./logo.svg"
 
-import { WithFeature } from '~/root'
+import { WithFeature } from "~/root"
 
```

Comment:
@shb46 I think if you revert this part of the change the error you are seeing will go away